### PR TITLE
March2022 training updates

### PR DESCRIPTION
**Pull request by reidchristopher**
Updates to training docs
- fix setup script Qt-Creator issues
- automatically setup Firefox bookmarks and home page in setup script
- fix to instructions for 4.1 (descartes)
- various clarity updates

### CODE REVIEW

**Comment by jdlangs** [comment]
Trying to version control the user Firefox profile seems messy and potentially brittle. I'd suggest making a desktop link to the specific page and making sure the bookmarks are linked to from there.

**Comment by reidchristopher** [comment]
Good point. Making favorites was simpler than making desktop icons (weird complications in allowing them to launch), but effectively accomplishes the same thing. This is the resulting side-bar

Screenshot from 2022-04-05 17-07-29

File: gh_pages/_source/session1/ros2/0-ROS-Setup.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ We believe we have a good installation of ROS but let's test it to make sure.
...
+In a "bare" ROS install, you will need to run the "source" command on **every** new shell you open to have access to the ROS commands.  If your project only uses a single distribution, it can be helpful to configure our `~/.bashrc` file to automatically source this setup file for each new terminal window.  See [here](https://docs.ros.org/en/foxy/Tutorials/Configuring-ROS2-Environment.html#add-sourcing-to-your-shell-startup-script) for details. If you will be using multiple ROS distributions on one machine, then you do NOT want this auto-sourcing behavior. The auto-sourcing will always source the same distrition, and sourcing another distribution after this can cause issues. This class uses multiple ROS distros (noetic for ROS1 and foxy for ROS2), so we have not configured this auto-sourcing behavior for your training PC.
```
Comment (jdlangs) [inline gh_pages/_source/session1/ros2/0-ROS-Setup.md]: typo (distrition)

**Review by jdlangs — APPROVED** [review]
Everything looks good on my end. Feel free to cleanup the commit history if you prefer or else we can just squash during the merge.

**Comment by jdlangs** [comment]
Nice job with the web page launchers, the icons are a nice touch too.

File: gh_pages/_source/session1/ros2/0-ROS-Setup.md (hunk 1 above)
Comment (reidchristopher) [reply gh_pages/_source/session1/ros2/0-ROS-Setup.md]: Good catch

**Comment by DavidMerzJr** [comment]
@jdlangs Is this ready to merge?

**Comment by jdlangs** [comment]
Yeah, looks good to me.

**Pull request merged**